### ISSUE TITLE
Mobile: Quickly/ multiple time changing the landscape/portrait mode on the iPhone results in no response from the device

### ISSUE THREAD

**Issue by amastej** [zebrunner/stf#875]
**Steps**
1. Open STF
2. Open iPhone 8 / iPhone 8 plus 
3. Quickly and multiple times change view from portrait to landscape 

**Actual result**
Quickly/ multiple time changing the landscape/portrait mode on the iPhone results in no response from the device or temporary blocking of the device
https://github.com/zebrunner/stf/assets/149781148/fd47ba2b-576c-4931-8deb-9f628b1afaa4
https://github.com/zebrunner/stf/assets/149781148/40d5b321-2d41-4477-8717-74f74f7df227

**Expected**
Quick, repeated changes to portrait/landscape modes have no negative effect on the device

**Comment by vdelendik** [comment]
@ignacionar, as I remember we lock operations already for swipe until previous command is finished. is it possible to the same for switching portrait mode?

**Comment by ignacionar** [comment]
@vdelendik Yes, I think we can do it, we could track if the device is rotating and block operations until its done

**Comment by vdelendik** [comment]
@dhreben, PR merged: https://github.com/zebrunner/stf/pull/881
building stf:3.6.3 to test and use as released artifact. please, make sure to test after final setup
